### ISSUE TITLE
checked Ctrl-Enter and it does something else  :-)

### ISSUE THREAD

**Issue by michaelsjackson** [JoepVanlier/Hackey-Patterns#42]
Now it adds where loop points are. But my suggestion should be as follows:

1. Taking only the length information from where the loop points are.
2. Adding should be where the cursor is.
So now the second point goes also to where the loop points are.

Wow, this was quick reply from you, was just writing the message, hit enter by accident. Then got already answer from you, cool.

Then we should also not need your already added ctrl-pgup/pgdn for moving the loop point, because it can stay and loop where it is, but we can still edit and add new stuff elsewhere. This should also fit better to the concept, listen somewhere, but edit elsewhere, all independently, not disturbing each other.

**Comment by JoepVanlier** [comment]
What does it do?

**Comment by JoepVanlier** [comment]
Ah ok. I'll fix it when I get home in a few hours.

**Comment by JoepVanlier** [comment]
2e62e9a4ec74e4bf9ede7036159f536c454848c0 ? :)

**Comment by michaelsjackson** [comment]
One small thing:
- if there is no range selected (using ctrl-q and ctrl-w) and I hit ctrl-enter, e.g. just after starting reaper, adding an item, starting hackey patterns, in my case nothing is selected yet, if I hit ctrl-enter, new patterns are added to the item/pattern list on the right, but not to the arrange. Not adding to the arrange is correct, but I guess it should also not add to the pattern list on the right, or?

Otherwise you can end up having multiple patterns with same name, e.g. 5 times a pattern named 01. But once you selected a range the naming is always changing, for each new pattern, how it should be.

In short again: Program should prevent adding new patterns if there is no range selection. Another variant could be, it would add a 1 bar selection to the beginning of the arrange, then adding new patterns as 1 bar empty patterns.
